<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ImageExportDialog</class>
 <widget class="QDialog" name="ImageExportDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>379</width>
    <height>220</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Little Navmap - Export Map as Image</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="0" colspan="2">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Help|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="labelResolution">
     <property name="text">
      <string>&amp;Resolution:</string>
     </property>
     <property name="buddy">
      <cstring>comboBoxResolution</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="comboBoxResolution">
     <property name="toolTip">
      <string>Select predefined image resolution in pixels. 
Note that all options except &quot;Current Map View&quot; will change visible details.</string>
     </property>
     <item>
      <property name="text">
       <string>Current Map View, %1 x %2</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>Custom Resolution</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>720p, 1280 × 720, 16∶9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1080p, 1920 × 1080, 16∶9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>1440p, 2560 × 1440, 16∶9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>2160p, 3840 × 2160, 16∶9</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>4320p, 7680 × 4320, 16:9</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QSpinBox" name="spinBoxHeight">
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>8192</number>
     </property>
     <property name="value">
      <number>1080</number>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="1">
    <widget class="QSpinBox" name="spinBoxWidth">
     <property name="showGroupSeparator" stdset="0">
      <bool>true</bool>
     </property>
     <property name="suffix">
      <string> pixel</string>
     </property>
     <property name="minimum">
      <number>32</number>
     </property>
     <property name="maximum">
      <number>8192</number>
     </property>
     <property name="value">
      <number>1920</number>
     </property>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="labelHeight">
     <property name="text">
      <string>&amp;Height:</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxHeight</cstring>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="labelWidth">
     <property name="text">
      <string>&amp;Width:</string>
     </property>
     <property name="buddy">
      <cstring>spinBoxWidth</cstring>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="2" alignment="Qt::AlignRight">
    <widget class="QCheckBox" name="checkBoxAvoidBlurred">
     <property name="toolTip">
      <string>Zooms the map out to get a sharper image. 
This will change the shown map region and might change details.</string>
     </property>
     <property name="text">
      <string>&amp;Zoom out to avoid blurred map</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>comboBoxResolution</tabstop>
  <tabstop>spinBoxWidth</tabstop>
  <tabstop>spinBoxHeight</tabstop>
  <tabstop>checkBoxAvoidBlurred</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
